CRM/Contribute/Form/Contribution/Main.php

```diff
@@ -479,7 +479,7 @@ public function buildQuickForm() {
     if (!($allAreBillingModeProcessors && !$this->_values['is_pay_later'])) {
       $submitButton = [
         'type' => 'upload',
-        'name' => CRM_Utils_Array::value('is_confirm_enabled', $this->_values) ? ts('Confirm Contribution') : ts('Contribute'),
+          'name' => CRM_Utils_Array::value('is_confirm_enabled', $this->_values) ? ts('Review your contribution') : ts('Contribute'),
```

Comment:
@kartik1000 can you fix the indent issue here?

Reply:
Yes, sure. 5 mins.